### PR TITLE
Allow Composer version to be configured via attribute

### PR DESCRIPTION
**Pull request by jcorme**
1.2.1 is really old.

### CODE REVIEW

**Comment by ramereth** [comment]
[CiviCRM website (Drupal)](https://trello.com/c/nOTMJqvz/3-civicrm-website-drupal)

File: attributes/default.rb [hunk 1]
```diff
@@ -4,3 +4,5 @@
 default['osl-php']['use_opcache'] = false
 # List PHP versions that require IUS archive repo: https://ius.io/LifeCycle/#php
 default['osl-php']['ius_archive_versions'] = %w(5.3 5.6 7.0)
+
+default['osl-php']['composer_version'] = '1.2.1'
```
Comment (ramereth) [inline attributes/default.rb]: Should we default this to the latest version instead?

File: attributes/default.rb [hunk 2]
```diff
@@ -4,3 +4,5 @@
 default['osl-php']['use_opcache'] = false
 # List PHP versions that require IUS archive repo: https://ius.io/LifeCycle/#php
 default['osl-php']['ius_archive_versions'] = %w(5.3 5.6 7.0)
+
```
Comment (ramereth) [inline attributes/default.rb]: Let's remove this whitespace please.

**Comment by ramereth** [comment]
@jcorme you still need to fix the InSpec test. Can you please do that?

**Comment by jcorme** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v4.0.3.  Have a nice day!

**Pull request merged**